packages/ui-toolkit/src/components/molecules/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -8,24 +8,42 @@ import './radioButtonGroup.css';
 
 const RadioButtonGroup = (props: Props) => {
 
-  const { radioButtons, containerClassName,
-    dataTestId, onSelect, selected, ...restProps } = props;
+  const {
+    radioButtons,
+    containerClassName,
+    dataTestId,
+    onSelect,
+    selected
+  } = props;
 
   return (
-    <div id="container"
+    <div
+      id="container"
       className={containerClassName}
     >
       {
         radioButtons.map((item: RadioButtonType, index: number) => {
+          const {
+            value,
+            parentClassName,
+            label,
+            size,
+            labelClassName,
+            radioDirection
+          } = item;
+
           return (
-            <div key={`${item.value}${index}`}
-              {...restProps}
```

Comment:
Why have we removed the restProps prop?

Reply:
That field would be empty always as we already have extracted all available fields in `Props` type. The changes has been verified in `js` files too where there could've been any extra props passed. 
cc: @AkshayNaikGroww